src/options.h

```diff
@@ -366,7 +366,7 @@ struct options_main {
 /* Vector width of OpenCL kernel */
 	unsigned int v_width;
 #endif
-#if defined(HAVE_OPENCL)
+#if defined(HAVE_OPENCL) || defined(HAVE_ZTEX)
 	struct list_main *gpu_devices;
```

Comment:
We should rename this from gpu_devices to something like `extra_devices` or `acc_devices` or something like that.

Since `git` blames someone named Claudio, I will talk to him myself. Also, in f72bbe558a25b727296af9e6ca98b58d70093906